picodom.d.ts

```diff
@@ -0,0 +1,26 @@
+export interface VNode<D> {
+  tag: string
+  data?: D
+  children: VElement<{}>[]
```

Comment:
So VElement just means `VNode | string` which is what you have there. It's an array of them which is also what you have there.